qless/__init__.py

```diff
@@ -129,6 +132,8 @@ def __init__(self, url='redis://localhost:6379', hostname=None, **kwargs):
         import socket
         # This is our unique idenitifier as a worker
         self.worker_name = hostname or socket.gethostname()
+        if PY3:
+            kwargs['decode_responses'] = True
```

Comment:
Yeah, decode away. As mentioned above, there is the requirement that the data be `json`-serializable already.